### PR TITLE
Addresses issue DCOS-632. Documentation Updated

### PR DESCRIPTION
**Pull request by rohithzr**
## What are your changes?
Added following steps that are required to make custom Auth0 Work.
1. Changes that need to be done in the custom login page.
2. Mentioned to use Custom Social Provider API Keys.

Hosted a page on github pages as a helper to generate the encoded config string

## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

@joel-hamill

### CODE REVIEW

**Comment by judithpatudith** [comment]
@joel-hamill A community member just submitted a docs patch, can you shepherd it in for them? Thank you!

**Comment by joel-hamill** [comment]
@alberts can you review this for technical accuracy?

**Comment by alberts** [comment]
lgtm. @jgehrcke for one last look.

File: 1.7/administration/id-and-access-mgt/configure-your-security.md [hunk 1]
```diff
@@ -14,29 +14,43 @@ to use an alternative Auth0 account by adding the following directive to
...
+```javascript
+var auth0 = new Auth0({
+   domain: "mydomain.auth0.com",
+   clientID: "myClientId",
+   callbackURL: "",
+   callbackOnLocationHash: true
+ });
+```
+
+If you don't setup your own social provider keys you will get an error saying "Social provider API keys are required"
```
Comment (alberts) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: full stop?

File: 1.7/administration/id-and-access-mgt/configure-your-security.md [hunk 2]
```diff
@@ -14,29 +14,43 @@ to use an alternative Auth0 account by adding the following directive to
...
 ```yaml
+oauth_enabled: 'true'
 oauth_issuer_url: https://youraccount.auth0.com/
 oauth_client_id: <client id from application>
 oauth_auth_redirector: https://youraccount.auth0.com
 oauth_auth_host: https://youraccount.auth0.com
 ```
 
-To obtain the client ID, complete the following steps:
+To obtain the client ID and using your own Auth0 account, complete the following steps:
```
Comment (jgehrcke) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: "... and to use your own ... "? (grammar)

File: 1.7/administration/id-and-access-mgt/configure-your-security.md [hunk 3]
```diff
@@ -14,29 +14,43 @@ to use an alternative Auth0 account by adding the following directive to
...
 1. Sign up for [Auth0](https://auth0.com/).
-2. Create a new application in your Auth0 dashboard.
+2. Create a new "Regular Web" application in your Auth0 dashboard.
 3. Skip the Quick Start documentation and switch to the Settings tab to obtain
    the client ID.
 4. Add `https://youraccount.auth0.com` to the Allowed Origins (CORS) setting.
 5. Show Advanced Settings at the bottom of the Settings page and change the
    JWT Signature Algorithm in the OAuth tab to RS256.
-6. Create a custom Login Page in Auth0, using the HTML/JavaScript source of
+6. Create a custom Login Page in Auth0 "Hosted Pages" Tab, using the HTML/JavaScript source of
```
Comment (jgehrcke) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: `in Auth0's "Hosted Pages" Tab`? (grammar)

**Review by jgehrcke — CHANGES_REQUESTED** [review]
Thanks, @Rohithzr -- this is nice to see. 

I have left two suggestions for grammar improvement. Would appreciate if you could go through them and modify the 1.7/1.8/1.9 docs equivalently.

Cheers

File: 1.7/administration/id-and-access-mgt/configure-your-security.md (hunk 2 above)
Comment (Rohithzr) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: @jgehrcke ok how does this look?

> To obtain the client ID and use your own Auth0 account, complete the following steps:

File: 1.7/administration/id-and-access-mgt/configure-your-security.md (hunk 3 above)
Comment (Rohithzr) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: @jgehrcke ok how does this look?

> In [Auth0's Hosted Pages](https://manage.auth0.com/#/login_page), Create a custom Login Page using the HTML/JavaScript source of...

File: 1.7/administration/id-and-access-mgt/configure-your-security.md (hunk 2 above)
Comment (jgehrcke) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: I think this looks great! :)

File: 1.7/administration/id-and-access-mgt/configure-your-security.md (hunk 3 above)
Comment (jgehrcke) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: Looks great! (but `Create` must be `create` then).

Comment (Rohithzr) [inline 1.7/administration/id-and-access-mgt/configure-your-security.md]: ok so i'll push the changes
Thanks

File: 1.8/administration/id-and-access-mgt/configure-your-security.md [hunk 4]
```diff
@@ -14,29 +14,43 @@ to use an alternative Auth0 account by adding the following directive to
 `genconf/config.yaml`:
 
 ```yaml
+oauth_enabled: 'true'
 oauth_issuer_url: https://youraccount.auth0.com/
 oauth_client_id: <client id from application>
 oauth_auth_redirector: https://youraccount.auth0.com
 oauth_auth_host: https://youraccount.auth0.com
 ```
```
Comment (Rohithzr) [inline 1.8/administration/id-and-access-mgt/configure-your-security.md]: done

File: 1.9/administration/id-and-access-mgt/configure-your-security.md [hunk 5]
```diff
@@ -14,28 +14,43 @@ to use an alternative Auth0 account by adding the following directive to
 `genconf/config.yaml`:
 
 ```yaml
+oauth_enabled: 'true'
 oauth_issuer_url: https://youraccount.auth0.com/
 oauth_client_id: <client id from application>
 oauth_auth_redirector: https://youraccount.auth0.com
 oauth_auth_host: https://youraccount.auth0.com
 ```
```
Comment (Rohithzr) [inline 1.9/administration/id-and-access-mgt/configure-your-security.md]: done

**Comment by Rohithzr** [comment]
@jgehrcke @joel-hamill @alberts I think I have implemented all your suggestions. Please verify the same.

Thanks.

**Review by joel-hamill — APPROVED** [review]

**Comment by Rohithzr** [comment]
@joel-hamill @jgehrcke this is ready to be merged, I hope.

**Review by jgehrcke — APPROVED** [review]

**Comment by jgehrcke** [comment]
@Rohithzr, thanks for the contribution!

**Comment by rohithzr** [comment]
There is no current documentation to achive this anymore, is changing the Auth0 client no longer possible?

**Pull request merged**